### PR TITLE
add cpe translations to release

### PR DESCRIPTION
**Pull request by michalnicp**
fleetdm/fleet#6628

### CODE REVIEW

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -33,23 +33,27 @@ jobs:
...
+        run: |
+          cd fleet
+          go mod download
+          go run -tags fts5 cmd/cpe/generate.go
+          cat ./etagenv >> $GITHUB_ENV
 
       # Note that a new commit must be made for each release, otherwise GitHub does not order
       # the releases properly.
       - name: Tag
-        if: hashFiles('fleet/*.sqlite.gz') != ''
```
Comment (roperzh) [inline .github/workflows/generate.yml:41]: just double checking: why did you decide to remove this? seems like this step will be now always run, independently if new files were generated or not?

**Review by roperzh — APPROVED** [review]

File: .github/workflows/generate.yml (hunk 1 above)
Comment (michalnicp) [reply .github/workflows/generate.yml:41]: This was redundant. If any previous step fails, then this step won't run. No need to override the if condition to check if the file was correctly generated.

Comment (roperzh) [reply .github/workflows/generate.yml:41]: ah, makes sense

File: .github/workflows/generate.yml [hunk 2]
```diff
@@ -33,23 +33,27 @@ jobs:
           go-version: '^1.17.3'
 
       - name: Generate CPE database
-        run: cd fleet && go mod download && go run -tags fts5 cmd/cpe/generate.go && cat ./etagenv >> $GITHUB_ENV
+        run: |
```
Comment (michalnicp) [inline .github/workflows/generate.yml:36]: Updated to be more readable. `|` preserves newlines. By default the shell is `/bin/bash -e` which means any errors cause an early exit so no need to `&&` everything together.

File: .github/workflows/generate.yml [hunk 3]
```diff
@@ -33,24 +33,32 @@ jobs:
...
+          cd fleet
+          go mod download
+          go run -tags fts5 cmd/cpe/generate.go
+
+      - name: Current date
```
Comment (michalnicp) [inline .github/workflows/generate.yml:41]: Use the current date for releases. If multiple releases are created on the same day, then github sorts in descending order by name. Using the etag (a hash) results in random order.

**Pull request merged**